helm-chart/templates/cvat_backend/server/deployment.yml

```diff
@@ -88,7 +88,15 @@ spec:
         {{- if .Values.cvat.backend.permissionFix.enabled }}
         - name: user-data-permission-fix
           image: busybox
-          command: ["/bin/chmod", "-R", "777", "/home/django"]
+          command: ["/bin/sh", "-c"]
```

Comment:
Could you create a helper function to reduce code cloning?

Reply:
Noticed the container definition is duplicated across deployments - I've refactored the whole block to `_helpers` instead of just the command property.